lib/topology/interact.py

```diff
@@ -53,6 +53,7 @@ class NamespaceCompleter(object):
     """
 
     def __init__(self, ns):
+        super(NamespaceCompleter, self).__init__()
```

Comment:
I remember our discussion... but what comes with calling the base `object` `__init__ `?